fpo-api/api/auth.py

```diff
@@ -146,6 +148,17 @@ def grecaptcha_verify(request) -> dict:
         "message": verify_rs.get("error-codes", None) or "Unspecified error.",
     }
 
+def get_email_service_token() -> {}:
+    client_id = settings.EMAIL_SERVICE_CLIENT_ID
+    client_secret = settings.EMAIL_SERVICE_CLIENT_SECRET
+    payload = {"grant_type":"client_credentials"}
+    header = {"content-type": "application/x-www-form-urlencoded"}
+    url = "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token"
```

Comment:
We're going to want this URL to be configurable like the associated ID and secret(s), since it will change depending on the deployment environment.

Reply:
I have made the changes for this.